index.js

```diff
@@ -1,10 +1,22 @@
 const execSync = require('child_process').execSync
 
 function getProcesses (command) {
+  const processesObject = {}
+
+  /* Sanitizes the input command by limiting the command name 
+   * to specific characters.
+   */
+  const regex = /[^A-Za-z0-9 .\-_]+/g
```

Comment:
The regexp doesn't seem to work with a potentially legit input:
the sample `"service".match(/[^A-Za-z0-9 .\-_]+/g)` return `null`.

Would you mind to provide further testing?